app/models/sale.rb

```diff
@@ -168,6 +169,12 @@ def accounts_receivable_reverse_event
     end
   end
 
+  def report_delivery(transition)
+    if transition.args[1] == '1'
```

Comment:
What does this mean?
Maybe a comment can be added to the code, describing why [1] have to be '1' etc.